setup.py

```diff
@@ -51,7 +51,10 @@ def local_file(name: str) -> Path:
     ),
     long_description_content_type="text/markdown",
     entry_points={
-        "flake8.extension": ["TRI = flake8_trio:Plugin"],
-        "console_scripts": ["flake8-trio=flake8_trio:main"],
+        # You're not allowed to register error codes longer than 3 characters. But flake8
+        # doesn't enforce anything about the characters trailing the code, so we can say
+        # the code is ASY and then just always happen to print NCxxx directly after it.
```

Comment:
😹

Reply:
(this has always been the case, but noticed now it's probably good to add a comment)